components/header.js

```diff
@@ -45,30 +24,30 @@ const Header = () => {
       </nav>
       <div className={pageStyles.rules}>
         <div className={pageStyles.cardRanks}>
-          <span>
+          <div className={pageStyles.navHeaders}>
             <h2>Card Ranks (High - Low)</h2>
-            <button className={headerRules.cardRank ? `${pageStyles.buttonCaret} ${pageStyles.divOpen}` : pageStyles.buttonCaret} onClick={handleCardRankClick}>
+            <button className={headerRules.cardRank ? `${pageStyles.buttonCaret} ${pageStyles.divOpen}` : pageStyles.buttonCaret} onClick={e => handleClick('cardRank')}>
```

Comment:
if you're not using the actual event itself, i think you can just write it like so

```suggestion
            <button className={headerRules.cardRank ? `${pageStyles.buttonCaret} ${pageStyles.divOpen}` : pageStyles.buttonCaret} onClick={() => handleClick('cardRank')}>
```

but let me know if that's wrong